spec/features/auth_user_cart_emptied_after_order_spec.rb

```diff
@@ -19,4 +19,29 @@
       visit checkout_path
       expect(page).to_not have_content("item1")
   end
+
+  it "all items are cleared after checkout" do
+      user = create(:default_user)
+      5.times do |x|
+        create(:item, title: "item#{x}")
+      end
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+      visit items_path
+      click_link "item1"
+      first(:button, "Add To Cart").click
+      click_link "item2"
+      first(:button, "Add To Cart").click
+      click_link "item3"
+      first(:button, "Add To Cart").click
+      visit checkout_path
+      expect(page).to have_content("item1")
+      click_button "Checkout"
+      click_button "Confirm Order"
+
+      visit checkout_path
+      expect(page).to_not have_content("item1") 
+      expect(page).to_not have_content("item2")
+      expect(page).to_not have_content("item3")
+    end
```

Comment:
`end` at 46, 4 is not aligned with `it "all items are cleared after checkout" do` at 23, 2